.github/workflows/pythonpackage.yml

```diff
@@ -24,7 +24,7 @@ jobs:
         cd bwa-0.7.17
         make -j$(nproc)
         cd ..
-        echo "::add-path::${GITHUB_WORKSPACE}/bwa-0.7.17/"
+        echo "${GITHUB_WORKSPACE}/bwa-0.7.17/" >> $GITHUB_PATH
```

Comment:
Needed because of this: https://github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands/